workspaces/ui-v2/src/entry-points/cloud/TopLevelRoutes.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { Route, Switch } from 'react-router-dom';
+import CloudViewer from '../../spectacle-implementations/cloud-viewer';
+import WelcomePage from '../../optic-components/onboarding/WelcomePage';
+
+export default function TopLevelRoutes() {
+  return (
+    <Switch>
+
+      <Route
+        strict
+        //@TODO: centralize this path pattern
```

Comment:
the intent of the comment here is to centralize all the root paths in an object map so it's more discoverable to developers

Reply:
This comment came with the copy-paste from `entry-points/local`. Is it not relevant here?

Reply:
no it's more relevant the more entry points there are. there should ideally be one place (like an object/map) where a developer can come and see the different entry points and ensure they don't collide, etc.